### PR TITLE
8.7.0-alpha2 release notes

### PR DESCRIPTION
**Pull request by mesellings**
## Description

8.7 alpha 2 release notes.

Based on [labeled epics](https://github.com/orgs/camunda/projects/9/views/31).

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [X] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [X] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by mesellings** [comment]
FYI @akeller

**Comment by mesellings** [comment]
Added late entries for [#2244](https://github.com/camunda/product-hub/issues/2244) and [#2245](https://github.com/camunda/product-hub/issues/2245)

**Comment by mesellings** [comment]
Removed https://github.com/camunda/product-hub/issues/1314 placeholder as this moved to alpha 3.

**Comment by mesellings** [comment]
Barring any further last-minute additions or removals, this is ready fro review @akeller 👍

**Comment by akeller** [comment]
In addition to reviewing myself, I also posted on #team-products for more general awareness - https://camunda.slack.com/archives/C02GZDWP45T/p1733499344248629

**Comment by mesellings** [comment]
> In addition to reviewing myself, I also posted on #team-products for more general awareness - https://camunda.slack.com/archives/C02GZDWP45T/p1733499344248629

Awesome, thank you - I've added this as a step on the canvas now as well 👍

**Comment by mesellings** [comment]
@berkaycanbc @akeller I've committed a new entry for the [Process Instance Migration work](https://camunda.slack.com/archives/C02GZDWP45T/p1733500069665329?thread_ts=1733499344.248629&cid=C02GZDWP45T) - please review.

image

**Review by akeller — DISMISSED** [review]
Nothing is jumping out at me, this looks good!

**Comment by akeller** [comment]
Monitor https://camunda.slack.com/archives/C03NFMH4KC6/p1733502438865109 for release status before merging and releasing.

**Review by berkaycanbc — DISMISSED** [review]
Process Instance Migration part LGTM 🚀

**Comment by mesellings** [comment]
@sbuettner I added in an entry for the testing migration work: 

image

**Comment by mesellings** [comment]
@sbuettner Webhook connector details updated 👍

**Review by sbuettner — DISMISSED** [review]
Awesome, thanks @mesellings

**Comment by mesellings** [comment]
@akeller Please can I get a final review, in anticipation of the release going ahead tomorrow morning?

File: docs/reference/release-notes/870.md
Comment (akeller) [inline docs/reference/release-notes/870.md:140]: Nitpick - I would remove the periods after each item in this list

File: docs/reference/release-notes/870.md
Comment (akeller) [inline docs/reference/release-notes/870.md:56]: ❓ Should we use the common abbreviation here (PII) for keyword search purposes?

**Review by akeller — APPROVED** [review]
Approving. I left a nit-picky comment and a question, but I don't consider either of those things blocking. AFAIK, this includes all work going into this alpha.

**Comment by mesellings** [comment]
Thanks @akeller, I've added those additional feedback edits into the next PR: https://github.com/camunda/camunda-docs/pull/4738

**Pull request merged**
